StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/USBankAccount/InstantDebitsPaymentMethodElement.swift

```diff
@@ -0,0 +1,219 @@
+//
+//  InstantDebitsPaymentMethodElement.swift
+//  StripePaymentSheet
+//
+//  Created by Krisjanis Gaidis on 4/11/24.
+//
+
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePayments
+@_spi(STP) import StripePaymentsUI
+@_spi(STP) import StripeUICore
+import UIKit
+
+final class InstantDebitsPaymentMethodElement: Element {
+
+    private let configuration: PaymentSheetFormFactoryConfig
+    private let formElement: FormElement
+
+    private var linkedBankElements: [Element] {
+        return [linkedBankInfoSectionElement]
+    }
+    private let linkedBankInfoSectionElement: SectionElement
+    private let linkedBankInfoView: BankAccountInfoView
+    private var linkedBank: InstantDebitsLinkedBank?
+    private let theme: ElementsUITheme
+    var presentingViewControllerDelegate: PresentingViewControllerDelegate?
+
+    var delegate: ElementDelegate?
+    var view: UIView {
+        return formElement.view
+    }
+    var mandateString: NSMutableAttributedString? {
+        var string: String?
+        if linkedBank != nil {
+            // TODO(kgaidis): localize
+            string = "By continuing, you agree to authorize payments pursuant to <terms>these terms</terms>."
```

Comment:
not the end state we want but its OK for now given the state of the project (still iterating on design, API's are private, feature behind a flag etc.)